### ISSUE TITLE
Implement ability to import/export pricing data from mSupply

### ISSUE THREAD

**Issue by katherine-sussol** [openmsupply/unified-codes#7]
### Description
Placeholder issue to discuss/plan out how we will handle pricing data (for drugs being bought/sold within mSupply) and the Universal Codes application.

### Goal
Consensus on approach. This will form the basis for separate sub-issues to build it out.

### Requirements
- Upgrade mSupply to submit reference prices
- Upgrade mSupply to use reference prices
- Add MSH data as an initial value for price if possible
- Include a timestamp with price data. This would give the ability to show price variation over time

### Current Discussion Points/Agreements
- Not required for #2
- Need agreement from current clients to use their data

**Comment by wlthomson** [comment]
Thanks for documenting this requirements, really helpful! I don't know too much about this area of functionality, but looks good to me!

I think this could be something to put into the MVP, but as this stage my thoughts are it might be something for after the initial core functionality is implemented and approved?

**Comment by katherine-sussol** [comment]
Yeah, probably not one to start early on until we have the core ironed out, as long as the schema can accomodate prices, then I think we should be ok?

**Comment by mark-prins** [comment]
yes; I'd leave out of MVP. it's important for sure, but a large job with a few moving parts. Best to leave it as a separate item of work, and to look at it after MVP.

**Comment by wlthomson** [comment]
Yeah, so I think concensus from all of us is that this functionality is something to keep in mind when working on the MVP, but not something we need to implement for it?

Re. the schema, I don't think prices will be too tricky 🤞 . Current thinking is to either add prices onto the schema just like other `property` nodes (would definitely need a `type` field, and perhaps additional fields for currency etc.), or to add a new node type just for prices (and maybe a new relation, e.g. `has_price`).

Something to be agreed later, but main point is that I think it should be relatively straightforward to extend the MVP schema to additional functionality like prices 👍.
